ur_robot_driver/include/ur_robot_driver/ros/hardware_interface.h

```diff
@@ -242,7 +242,8 @@ class HardwareInterface : public hardware_interface::RobotHW
   std::vector<std::string> joint_names_;
   int32_t robot_mode_;
   int32_t safety_mode_;
-
+  std::array<double, 6> output_double_register_;
+  
```

Comment:
This line is not needed. See my comment. You should be able to just use `joint_efforts_`, which is already a member of this class.